spacy/structs.pxd

```diff
@@ -75,6 +75,7 @@ cdef struct TokenC:
     uint32_t r_edge
 
     int sent_start
+    int sent_end
```

Comment:
We shouldn't add this to the struct, that's a waste of space and it's data duplication that's hard to keep in-sync with the `sent_start` attribute. It's okay to have `sent_end` as an attribute, but it has to be computed dynamically by checking whether the following token is `sent_start`.